miso-web/src/main/webapp/scripts/workflow_display.js

```diff
@@ -0,0 +1,167 @@
+WorkflowDisplay = (function() {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "WorkflowDisplay" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)